mmv1/third_party/terraform/tests/resource_container_node_pool_test.go.erb

```diff
@@ -1594,6 +1620,92 @@ resource "google_container_node_pool" "with_linux_node_config" {
 }
 <% end -%>
 
+
+<% unless  version == 'ga' -%>
+func testAccContainerNodePool_withNetworkConfig(cluster, np, network string) string {
+	return fmt.Sprintf(`
+resource "google_compute_network" "container_network" {
+  name                    = "%s"
+  auto_create_subnetworks = false
+}
+
+resource "google_compute_subnetwork" "container_subnetwork" {
+  name                     = google_compute_network.container_network.name
+  network                  = google_compute_network.container_network.name
+  ip_cidr_range            = "10.0.36.0/24"
+  region                   = "us-central1"
+  private_ip_google_access = true
+
+  secondary_ip_range {
+    range_name    = "pod"
+    ip_cidr_range = "10.0.0.0/19"
+  }
+
+  secondary_ip_range {
+    range_name    = "svc"
+    ip_cidr_range = "10.0.32.0/22"
+  }
+
+  secondary_ip_range {
+    range_name    = "another-pod"
+    ip_cidr_range = "10.1.32.0/22"
+  }
```

Comment:
I'm still seeing the test fail due to adding a new secondary_ip_range here. Lets add an `ignore_changes` on secondary_ip_range to prevent that, and add a comment that provisioning a node pool with `network_config` defined may modify the subnetwork it exists in.